lib/src/utils/w3_header_utils.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class W3HeaderUtils {
```

Comment:
Let's use the actual name and change W3* members to W3C*  for all of the file content.